### DISCUSSION TITLE
Downloading onnx file

### DISCUSSION THREAD

**Discussion by aamir-gmail** [infiniflow/ragflow#2645] [General]
Each time the Ragflow server starts  in docker-compose  it download onnx file see logs ".gitattributes: 100%|██████████| 1.52k/1.52k [00:00<00:00, 8.70MB/s]
layout.onnx: 100%|██████████| 12.2M/12.2M [00:00<00:00, 82.1MB/s]
README.md: 100%|██████████| 28.0/28.0 [00:00<00:00, 154kB/s]MB/s]
rec.onnx: 100%|██████████| 10.8M/10.8M [00:00<00:00, 77.5MB/s]/s]
det.onnx: 100%|██████████| 4.75M/4.75M [00:00<00:00, 19.3MB/s]
layout.laws.onnx: 100%|██████████| 12.2M/12.2M [00:00<00:00, 32.0MB/s]
ocr.res: 100%|██████████| 26.2k/26.2k [00:00<00:00, 69.9MB/s]t/s]
tsr.onnx: 100%|██████████| 12.2M/12.2M [00:00<00:00, 53.4MB/s]9.6MB/s]
layout.paper.onnx: 100%|██████████| 12.2M/12.2M [00:00<00:00, 50.3MB/s]
layout.manual.onnx: 100%|██████████| 12.2M/12.2M [00:00<00:00, 50.9MB/s]
Fetching 10 files: 100%|██████████| 10/10 [00:02<00:00,  4.31it/s]0MB/s]
ragflow-server  | 

can we persist these files in a volume so it is not done again, or pack them up in the image

**Comment by KevinHuSh** [comment]
Try this image: infiniflow/ragflow:dev, which has already downloaded all these model.

**Comment by aamir-gmail** [comment]
Thanks, I'll check it out.

On Sun, Sep 29, 2024 at 12:49 PM Kevin Hu ***@***.***> wrote:

> Try this image: infiniflow/ragflow:dev, which has already downloaded all
> these model.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/infiniflow/discussions/2645#discussioncomment-10786776>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJA2ECPRPE2HZRSB7AL23LLZY5TFBAVCNFSM6AAAAABPBEGJLSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZYGY3TONQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Kind Regards


Aamir Mirza
